addons/hr/models/res_users.py

```diff
@@ -183,22 +183,13 @@ def write(self, vals):
             raise AccessError(_("You are only allowed to update your preferences. Please contact a HR officer to update other information."))
 
         result = super(User, self).write(vals)
-
         employee_values = {}
         for fname in [f for f in ['name', 'email', 'image_1920', 'tz'] if f in vals]:
             employee_values[fname] = vals[fname]
         if employee_values:
             if 'email' in employee_values:
                 employee_values['work_email'] = employee_values.pop('email')
-            if 'image_1920' in vals:
-                without_image = self.env['hr.employee'].sudo().search([('user_id', 'in', self.ids), ('image_1920', '=', False)])
-                with_image = self.env['hr.employee'].sudo().search([('user_id', 'in', self.ids), ('image_1920', '!=', False)])
-                without_image.write(employee_values)
-                if not can_edit_self:
-                    employee_values.pop('image_1920')
-                with_image.write(employee_values)
-            else:
-                self.env['hr.employee'].sudo().search([('user_id', 'in', self.ids)]).write(employee_values)
```

Comment:
Similar question here, why change this logic about sync between user and employee?

Reply:
My reasoning for removing this is that now there shouldn't be a differentiation on employees without image or with image, since if no image is set they will show the avatar. Or do you mean the logic about `can_edit self`?

Reply:
I'm not removing the 1920 fields, so the value is being written to the hr.employee anyway. The problem here is that I removed the `not can_edit_self` check?